docs/configuration/OVERVIEW.md

```diff
@@ -36,103 +35,152 @@ A component will have to handle different types of configurations.
 
 ### Default configuration
 
-- Each __component type__ will have a default configuration which will be defined in _.config.ts_ file associated to the
-  component
+- Each __configurable component__ will have a default configuration which will be defined in the `.config.ts` file associated to the component
 
-> __WARNING__ the field name 'id' should not be used in the configuration, as we created a unique one for the entity configuration store
+> [!WARNING]
+> The field name `id` should not be used in the configuration, as we created a unique one for the entity configuration store.
 
-### Configuration coming from config store
+### Configuration coming from the configuration store
 
-- Each __component type__ can have a customized config (compliant with the store model). This custom config can be
+- Each __configurable component__ can have a customized configuration (compliant with the store model). This custom configuration can be
```

Comment:
This is a bit strange to phrase it this way and call it store configuration in the rest of the documentation. 
Maybe we could call it runtime configuration and only mention the store as a way to store this configuration instead of making it seems like the update of the store is the purpose of the configuration (not just the way the configuration is stored in the application).